test/doctest/mathjax-cdn.html

```diff
@@ -1,5 +1,5 @@
 <!-- .mathjax-cdn -->
-<link href="https://cdn.jsdelivr.net/npm/reveal.js@3.9.2/css/reset.css" rel="stylesheet"><link href="https://cdn.jsdelivr.net/npm/reveal.js@3.9.2/css/reveal.css" rel="stylesheet"><link href="https://cdn.jsdelivr.net/npm/reveal.js@3.9.2/css/theme/black.css" id="theme" rel="stylesheet">
+<link href="https://cdn.jsdelivr.net/npm/reveal.js@4.1.1/dist/reset.css" rel="stylesheet"><link href="https://cdn.jsdelivr.net/npm/reveal.js@4.1.1/dist/reveal.css" rel="stylesheet"><link href="https://cdn.jsdelivr.net/npm/reveal.js@4.1.1/dist/theme/black.css" id="theme" rel="stylesheet">
```

Comment:
We should update the attribute `:revealjsdir:` in `mathjax-cdn.adoc` and update the expectations:

```
:revealjsdir: https://cdn.jsdelivr.net/npm/reveal.js@4.0.2
```

```suggestion
<link href="https://cdn.jsdelivr.net/npm/reveal.js@4.0.2/dist/reset.css" rel="stylesheet"><link href="https://cdn.jsdelivr.net/npm/reveal.js@4.0.2/dist/reveal.css" rel="stylesheet"><link href="https://cdn.jsdelivr.net/npm/reveal.js@4.0.2/dist/theme/black.css" id="theme" rel="stylesheet">
```